gnocchi/tests/functional/gabbits/base.yaml

```diff
@@ -132,6 +132,7 @@ tests:
     authorization: "basic YWRtaW46"
   response_json_paths:
     $.storage.`len`: 2
+    $.metricd.`len`: 1
```

Comment:
should this be metricd.processors.`len`?

Reply:
No, because the len of `metricd.processors` depend on the number of workers and in this case depends on the coordination driver, which is likely to be e.g. SQL to it'd be NULL.

I"m just checking the key `processors` is here, like we do L134.